backend/src/nodes/properties/outputs/pytorch_outputs.py

```diff
@@ -1,10 +1,51 @@
-from .. import expression
+try:
+    import torch
+except:
+    torch = None
```

Comment:
I hate this. Just remove `from .pytorch_outputs import *` from `__init__.py` and we should be able to make them all static imports.

Man, I hate that python imports suck so much.

Reply:
Same